crates/bitcoin-da/src/verifier.rs

```diff
@@ -1116,30 +1098,6 @@ mod tests {
         );
     }
 
-    #[test]
```

Comment:
why remove test

Reply:
if you check closely, you will see that it is the exact same test with: break_completeness_proof_order test

Reply:
let's remove `txs.swap(2, 3);` from break_completeness_proof_order and restore this test then